### DISCUSSION TITLE
Email and Display Settings in Workspace Settings

### DISCUSSION THREAD

**Discussion by t-AIR-e** [makeplane/plane#2549] [Ideas]
I'd like the ability to modify email settings directly within a workspace on plane.so, rather than solely depending on the ENV. I've struggled to configure the SMTP settings on my local plane.so instance. Incorporating an in-app SMTP settings editor with a "test connection" feature would be awesome.

And for any other settings that are somewhere else I think consolidating all the defaults in the workspace settings tab would be great to have or at least the defaults for my profile.

To enhance user experience, consolidating all settings, including any that may be scattered elsewhere, into the workspace settings tab would be ideal. Having a unified location for all default configurations, especially for profile-specific ones, would greatly improve accessibility and ease of use.

**Comment by 0x0factor** [comment]
Submitting feedback and feature requests directly to Plane.so's development team or through their support channels would be the most effective way to communicate your ideas. Here's what you can consider doing:

Check Documentation:
Before providing feedback, check the official Plane.so documentation or support resources to see if there are any existing features or solutions for the issues you're facing.

Submit Feedback Through the Platform:
Many services have a dedicated feedback or support channel. This could be a feedback form on their website, an email address, or a community forum. Look for such channels on the Plane.so platform and submit your suggestions there.

Use the Platform's Support Channels:
If there's a support system in place, consider reaching out through their support channels to discuss your difficulties with configuring SMTP settings and any other challenges you're facing. They might be able to provide guidance or consider your suggestions for future updates.

Feature Request:
Clearly outline your feature request, explaining the benefits and how it could enhance user experience. Provide specific details about the in-app SMTP settings editor, the "test connection" feature, and the consolidation of settings into the workspace settings tab.

Engage with the Community:
If Plane.so has a community forum or discussion platform, engage with the community. Others might share similar concerns or have valuable insights into potential solutions.

Stay Informed:
Keep an eye on Plane.so's release notes and announcements. They may already be working on improvements, and your suggestions could align with their future plans.

**Reply by t-AIR-e** [reply]
OK, ChatGPT?
